Makefile.am

```diff
@@ -49,11 +49,11 @@ pkgconfigdir = @pkgconfigdir@
 # to build src/mpi/errhan/defmsg.h
 errnames_txt_files = 
 
-external_subdirs = @mplsrcdir@ @zmsrcdir@ @hwlocsrcdir@ @jsonsrcdir@
-external_ldflags = @mpllibdir@ @zmlibdir@ @hwloclibdir@ @netloclibdir@
```

Comment:
I genuinely don't know here so I'm bringing in @jain-surabhi-23. Do we not need izem after swapping out OPA with MPL?

Reply:
We dont use OPA or izem atomics anymore. We have moved to MPL in the past. So we are good.